tests/helpers.py

```diff
@@ -369,9 +374,12 @@ def get_cephblockpool_name():
     return pool_list
 
 
-def delete_cephblockpool():
+def delete_cephblockpool(cbp_name):
     """
-    Function for deleting CephBlockPool
+    Function for deleting specific CephBlockPool
+
+    Args:
+        cbp_name (str): Name of cbp for deletion
```

Comment:
s/cbp/CBP - if we are going to abbreviate let's capitalize (in docstrings)

Reply:
Done